CHANGELOG.md

```diff
@@ -13,7 +13,13 @@ We use the following categories for changes:
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
-## [Unreleased]
+## [0.11.1]
+
+### Fixed
+- Trace query returns empty result when queried with
+  - Tags from process table in Jaeger UI [#1385]
+  - Tags that have a numeric value, like `http.status_code=200` [#1385]
+  - Tags that involve status code [#1384]
```

Comment:
Few more things to add:
```
  - Support arrays as trace attribute values [#1381]
  - Fix race condition when stopping ingest [#1370]
  - Add HEAD and OPTIONS methods for `/healthz` endpoint [#1378]
```

Reply:
Please add those things to your PRs in the future. This PR is only doing cherry-pick, so if those were in code, it would appear here too.

Reply:
Added.